### PR TITLE
updating variables

### PR DESCRIPTION
**Pull request by iruzevic**

### CODE REVIEW

File: blocks/init/src/Blocks/components/image/components/image-options.js [hunk 1]
```diff
@@ -68,15 +60,15 @@ export const ImageOptions = (attributes) => {
 							label={
 								<>
 									<Icon icon={icons.link} />
-									{__('Image', 'eightshift-frontend-libs')}
+									{__('Image Url', 'Redesign')}
```
Comment (dingo-d) [inline blocks/init/src/Blocks/components/image/components/image-options.js:63]:
```suggestion
									{__('Image Url', 'eightshift-frontend-libs')}
```

File: blocks/init/src/Blocks/components/image/components/image-options.js [hunk 2]
```diff
@@ -86,7 +78,7 @@ export const ImageOptions = (attributes) => {
 								return (
 									<BaseControl
 										key={keyName}
-										label={sprintf(__('Image %s screen size', 'eightshift-frontend-libs'), pointLabel)}
+										label={sprintf(__('Image %s screen size', 'Redesign'), pointLabel)}
```
Comment (dingo-d) [inline blocks/init/src/Blocks/components/image/components/image-options.js:81]:
```suggestion
										label={sprintf(__('Image %s screen size', 'eightshift-frontend-libs'), pointLabel)}
```

File: blocks/init/src/Blocks/components/image/components/image-options.js [hunk 3]
```diff
@@ -97,7 +89,7 @@ export const ImageOptions = (attributes) => {
 													className={'custom-full-width-btn'}
 													onClick={() => setAttributes({ [attr]: {} })}
 												>
-													{sprintf(__('Remove %s screen size image', 'eightshift-frontend-libs'), pointLabel)}
+													{sprintf(__('Remove %s screen size image', 'Redesign'), pointLabel)}
```
Comment (dingo-d) [inline blocks/init/src/Blocks/components/image/components/image-options.js:92]:
```suggestion
													{sprintf(__('Remove %s screen size image', 'eightshift-frontend-libs'), pointLabel)}
```

**Review by dingo-d — CHANGES_REQUESTED** [review]
Some minor copy/paste typos

**Review by dingo-d — APPROVED** [review]

**Review by volfkarlo — APPROVED** [review]
Awesome job, so happy about these changes!

File: blocks/init/src/Blocks/assets/styles/editor/_editor.scss [hunk 4]
```diff
@@ -1,23 +1,16 @@
...
-);
-
 body .editor-styles-wrapper {
 	@include font-smoothing;
-	font-family: map-get-strict($override-map, font-family);
-	font-size: map-get-strict($override-map, font-size);
-	color: map-get-strict($override-map, color);
-	background-color: map-get-strict($override-map, background-color);
+	font-family: var(--global-font-family);
+	font-size: var(--base-font-size);
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply blocks/init/src/Blocks/assets/styles/editor/_editor.scss:7]: I will need to test it

File: blocks/init/src/Blocks/components/accordion/accordion-style.scss [hunk 5]
```diff
@@ -1,12 +1,9 @@
-$accordion: (
-	border-color: global-settings(colors, light),
-	icon-size: 1.5em,
-);
-
 .accordion {
+	--accordion-scoped-icon-size: 1.5em;
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply blocks/init/src/Blocks/components/accordion/accordion-style.scss:2]: I will test this and check

File: blocks/init/src/Blocks/components/loader/loader-style.scss [hunk 6]
```diff
@@ -27,8 +24,8 @@ $loader: (
 		z-index: 2;
 
 		&--1 {
-			border-right-color: map-get-strict($loader, item1);
-			border-top-color: map-get-strict($loader, item1);
+			border-right-color: var(--loader-scoped-item-1-color);
+			border-top-color: var(--loader-scoped-item-1-color);
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply blocks/init/src/Blocks/components/loader/loader-style.scss:28]: there is really no need for the fallback. It is scoped variable that implements the global color. If the global color is not there it the fallback will also fail

**Comment by iruzevic** [comment]
I will check all the issues here and fix them in the next PR

**Pull request merged**